docs/resources/azurerm_ad_user.md

```diff
@@ -227,7 +229,7 @@ requests are always welcome.
 
 This InSpec audit resource has the following special matchers. For a full list of
 available matchers, please visit our [Universal Matchers
-page](https://www.inspec.io/docs/reference/matchers/).
+page](/inspec/matchers/).
```

Comment:
Changing this to a relative link will break it for users looking directly at the resource pack docs folder.  Could this be updated to the full link?